benchmarking/repos/hg.py

```diff
@@ -55,13 +55,14 @@ def getCommitTime(self, commit):
         end = t.index('<END>')
         return int(float(t[start:end]))
 
-    def getNextCommitHash(self, commit):
-        # always get the top-of-trunk commit
-        c = self._run('log', '--template', '<START>{node}<END>',
-                      '-r', 'tip')
-        start = c.index('<START>') + len('<START>')
-        end = c.index('<END>')
-        return c[start:end]
+    def getNextCommitHash(self, commit, step):
+        # always get the top-of-trunk commit if there is no more commits
```

Comment:
remove comment since it is no longer true

Reply:
I think i changed the comments. :) If there is no more commits, it indeed use the top-of-trunk.